spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/interpreter/DynamicInterpreter.scala

```diff
@@ -188,20 +190,21 @@ object DynamicInterpreter {
     * Returns an interpreter for a mapping rule based on which strategy is applicable.
     *
     * @param rule   A conformance rule.
-    * @param schema A schema of a DataFrame to be conformed.
     * @return A mapping rule interpreter.
     */
-  private def getMappingRuleInterpreter(rule: MappingConformanceRule,
-                                        schema: StructType)
+  private def getMappingRuleInterpreter(rule: MappingConformanceRule)
                                        (implicit ictx: InterpreterContext): RuleInterpreter = {
-    if (ictx.featureSwitches.experimentalMappingRuleEnabled) {
-      if (canMappingRuleBroadcast(rule, schema)) {
-        MappingRuleInterpreterBroadcast(rule, ictx.conformance)
-      } else {
+    if (canMappingRuleBroadcast(rule)) {
+      log.info("Broadcast strategy for mapping rules is used")
```

Comment:
I wonder if he log level `info` is appropriate here. Wouldn't `debug` be more suitable?

Reply:
I'm more inclined towards `info` since it is possible that the choice of the strategy affects the results. So this information is useful in prod logs.

Reply:
Fair enough